app/services/rebate_forms_service.rb

```diff
@@ -5,6 +5,28 @@ def initialize(rebate_form_attributes)
     @rebate_form_attributes = rebate_form_attributes
   end
 
+  def generate_qr(rebate_form)
+    # create a new token that only allows you to fetch a restricted set of details for this application only
+    # and use the same token to submit the signatures back
+    # this token is valid for 30 minutes
+
+    payload = {
+      rebate_form_id: rebate_form.id,
+      exp: Time.now.to_i + (1000 * 60),
+      per: 'fetch_application_and_submit_signatures'
```

Comment:
Probably a good idea to take the value of `per` as a parameter. Makes this code more reusable, especially if it gets moved into a `JwtService`